core/src/test/scala/org/apache/spark/launcher/LauncherBackendSuite.scala

```diff
@@ -34,6 +35,8 @@ class LauncherBackendSuite extends SparkFunSuite with Matchers {
     "standalone/client" -> "local-cluster[1,1,1024]")
 
   tests.foreach { case (name, master) =>
+    // The tests here are failed due to the cmd length limitation up to 8K on Windows.
+    assume(!Utils.isWindows)
```

Comment:
This test is being failed due to cmd length limitation on Windows up to 8K. The actual command is roughly 16K (see https://gist.github.com/HyukjinKwon/d3fdd2e694e5c022992838a618a516bd)